services/gitdiff/csv.go

```diff
@@ -161,31 +161,6 @@ func createCsvDiff(diffFile *DiffFile, baseReader *csv.Reader, headReader *csv.R
 	createDiffRow := func(aline int, bline int) (*TableDiffRow, error) {
 		cells := make([]*TableDiffCell, columns)
 
-		if aline == 0 || bline == 0 {
```

Comment:
As mentioned in the PR comment, this is not needed as the code below for all rows works